analytical/tests/test_tag_piwik.py

```diff
@@ -145,3 +145,8 @@ def test_disable_usertrack(self):
         msg = 'Incorrect Piwik user tracking rendering.\nFound:\n%s\nIn:\n%s'
         var_code = '_paq.push(["setUserId", "BDFL"]);'
         self.assertNotIn(var_code, r, msg % (var_code, r))
+
+    @override_settings(PIWIK_DISABLE_COOKIES=True)
+    def test_disable_coolies(self):
```

Comment:
and I managed to do a typo here :) Should be `test_disable_cookies`. Sorry about that.

Reply:
Well, this typo is actually cool! Haha

Functionally, the test is still relevant and correct, I would suppose.

Reply:
Of course, it's just the test name.

Reply:
Fixed in ba89b80 :-)